src/test/java/no/ssb/jsontemplate/valueproducer/UuidValueProducerTest.java

```diff
@@ -9,8 +10,15 @@
 import static org.hamcrest.Matchers.*;
 import static org.hamcrest.Matchers.is;
 
-public class UuidValueProducerTest {
-    private UuidValueProducer producer = new UuidValueProducer();
+class UuidValueProducerTest {
+    private final UuidValueProducer producer = new UuidValueProducer();
+
+    @Test
+    @DisplayName("generates a random UUID string")
+    void testTypeName() {
+        String typeName = producer.getTypeName();
+        Assertions.assertEquals("uuid", typeName);
```

Comment:
Would be nice to still use the hamcrest matcher (`assertThat`), to follow the pattern of other tests.

Reply:
✅  Changed